agent/Makefile

```diff
@@ -213,83 +224,71 @@ tool-postprocess = \
+	${COPY} stockpile ${DESTDIR}
 
-install-dirs:
+install-destdir:
 	${INSTALL} ${INSTALLOPTS} ${DESTDIR}
-	${INSTALL} ${INSTALLOPTS} ${DESTDIR}/ansible
-	${INSTALL} ${INSTALLOPTS} ${CONFIGDIR}
-	${INSTALL} ${INSTALLOPTS} ${CONFIGDIR}/benchmark
-	${INSTALL} ${INSTALLOPTS} ${UTILDIR}
-	${INSTALL} ${INSTALLOPTS} ${BENCHDIR}
-	${INSTALL} ${INSTALLOPTS} ${BENCHDIR}/postprocess
-	${INSTALL} ${INSTALLOPTS} ${BENCHDIR}/templates
-	${INSTALL} ${INSTALLOPTS} ${TOOLDIR}
-	${INSTALL} ${INSTALLOPTS} ${TOOLDIR}/postprocess
-	${INSTALL} ${INSTALLOPTS} ${TOOLDIR}/datalog
-	${INSTALL} ${INSTALLOPTS} ${LIBDIR}
-	${INSTALL} ${INSTALLOPTS} ${DESTDIR}/stockpile
 
-install-ansible:
-	${COPY} -r ansible ${DESTDIR}
+install-lib:
+	${INSTALL} ${INSTALLOPTS} ${LIBDIR}
+	${COPY} lib ${DESTDIR}
 
-install-config:
-	${COPY} -r config ${DESTDIR}
+install-python3-setup: install-util-scripts install-lib
+	mkdir -p ${DESTDIR}/python3
+	(cd ..; SKIP_GENERATE_AUTHORS=1 SKIP_WRITE_GIT_CHANGELOG=1 python3 setup.py install --prefix=${DESTDIR}/python3)
+	${COPY} ${DESTDIR}/python3/bin/pbench-config ${UTILDIR}/
+	rm -rf ${DESTDIR}/python3
+	${COPY} ../lib/pbench ${LIBDIR}/
+	rm -r $$(find ${LIBDIR} -name __pycache__) ${LIBDIR}/pbench/test ${LIBDIR}/pbench/server ${LIBDIR}/pbench/cli/server
 
-install-util-scripts:
+install-util-scripts: install-destdir
+	${INSTALL} ${INSTALLOPTS} ${UTILDIR}
 	cd util-scripts; \
 	${COPY} ${util-scripts} ${UTILDIR}
 
-install-bench-scripts:
+install-bench-scripts: install-destdir
+	${INSTALL} ${INSTALLOPTS} ${BENCHDIR}
+	${INSTALL} ${INSTALLOPTS} ${BENCHDIR}/templates
 	cd bench-scripts; \
 	${COPY} ${bench-scripts} ${BENCHDIR}
+	${INSTALL} ${INSTALLOPTS} ${BENCHDIR}/postprocess
 	cd bench-scripts/postprocess; \
 	${COPY} ${bench-postprocess} ${BENCHDIR}/postprocess
 	cd ${BENCHDIR}; \
 	ln -sf postprocess/compare-bench-results compare-bench-results
 
-# install-bench-script-postprocess:
-# 	cd bench-scripts; \
-# 	${COPY} -r postprocess ${BENCHDIR}
-
-install-tool-scripts:
+install-tool-scripts: install-destdir
+	${INSTALL} ${INSTALLOPTS} ${TOOLDIR}
 	cd tool-scripts; \
 	${COPY} ${tool-scripts} ${TOOLDIR}
+	${INSTALL} ${INSTALLOPTS} ${TOOLDIR}/datalog
 	cd tool-scripts/datalog; \
 	${COPY} ${tool-datalogs} ${TOOLDIR}/datalog
+	${INSTALL} ${INSTALLOPTS} ${TOOLDIR}/postprocess
 	cd tool-scripts/postprocess; \
 	${COPY} ${tool-postprocess} ${TOOLDIR}/postprocess
 
-install-stockpile:
-	${COPY} -r stockpile ${DESTDIR}
-
-install-lib:
-	${COPY} -r lib ${DESTDIR}
-
-# Yuck!
-install-configtools:
-	if [ ! -d lib/pbench ] ;then ${COPY} -r ../lib/pbench ${LIBDIR} ;fi
-
-# SHA1 and SEQNO - these are used when building an RPM only
-# so we provide a target for the spec file to invoke. This
-# is *NOT* meant to be invoked interactively.
-install-build-artifacts:
-	${COPY} SHA1 ${DESTDIR}
-	${COPY} SEQNO ${DESTDIR}
+install-ansible: install-destdir
+	${INSTALL} ${INSTALLOPTS} ${ANSIBLEDIR}
+	cd ansible; \
+	${COPY} ara ${ANSIBLEDIR}
```

Comment:
Only the `ara` directory? Huh.

Reply:
Yes, we generally don't need any ansible stuff to be installed with two exceptions: the ara collector and pbench-run-benchmark (which is stored elsewhere).